[2024-09-11T04:03:55.613+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-11T04:03:55.632+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Filter_data.query_data_from_mongo manual__2024-09-11T04:03:54.073778+00:00 [queued]>
[2024-09-11T04:03:55.641+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Filter_data.query_data_from_mongo manual__2024-09-11T04:03:54.073778+00:00 [queued]>
[2024-09-11T04:03:55.642+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-09-11T04:03:55.657+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): query_data_from_mongo> on 2024-09-11 04:03:54.073778+00:00
[2024-09-11T04:03:55.666+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5365) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T04:03:55.668+0000] {standard_task_runner.py:72} INFO - Started process 5367 to run task
[2024-09-11T04:03:55.666+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Filter_data', 'query_data_from_mongo', 'manual__2024-09-11T04:03:54.073778+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/data-prep-for-model.py', '--cfg-path', '/tmp/tmpl7rft3xw']
[2024-09-11T04:03:55.669+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask query_data_from_mongo
[2024-09-11T04:03:55.714+0000] {task_command.py:467} INFO - Running <TaskInstance: Filter_data.query_data_from_mongo manual__2024-09-11T04:03:54.073778+00:00 [running]> on host dca6f81c3bd4
[2024-09-11T04:03:55.801+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Filter_data' AIRFLOW_CTX_TASK_ID='query_data_from_mongo' AIRFLOW_CTX_EXECUTION_DATE='2024-09-11T04:03:54.073778+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-11T04:03:54.073778+00:00'
[2024-09-11T04:03:55.802+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-11T04:03:55.859+0000] {transport.py:1923} INFO - Connected (version 2.0, client OpenSSH_8.2p1)
[2024-09-11T04:03:55.908+0000] {transport.py:1923} INFO - Authentication (password) successful!
[2024-09-11T04:03:59.217+0000] {python.py:240} INFO - Done. Returned value was: /opt/***/data/mongo_data.json
[2024-09-11T04:03:59.246+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-11T04:03:59.247+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Filter_data, task_id=query_data_from_mongo, run_id=manual__2024-09-11T04:03:54.073778+00:00, execution_date=20240911T040354, start_date=20240911T040355, end_date=20240911T040359
[2024-09-11T04:03:59.297+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-11T04:03:59.341+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-11T04:03:59.344+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
